From 988c708cb76076829f067d85e227480d97fc2d25 Mon Sep 17 00:00:00 2001
From: James Hughes <JamesH65@users.noreply.github.com>
Date: Tue, 20 Feb 2018 13:05:21 +0000
Subject: [PATCH 225/454] Update rpi-ft5406-overlay.dts

The Y resolution of the touchscreen was incorrectly set to 600, should be 480.
---
 arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
@@ -12,7 +12,7 @@
 				firmware = <&firmware>;
 				status = "okay";
 				touchscreen-size-x = <800>;
-				touchscreen-size-y = <600>;
+				touchscreen-size-y = <480>;
 				touchscreen-inverted-x = <0>;
 				touchscreen-inverted-y = <0>;
 				touchscreen-swapped-x-y = <0>;
